src/Components/ScrivitoExtensions/IconEditorTab/AllIcons.js

```diff
@@ -2,23 +2,23 @@ import * as React from "react";
 import { take } from "lodash-es";
 import fontAwesomeIcons from "./fontAwesomeIcons";
 
+const categoryMap = {};
+fontAwesomeIcons.forEach((icon) =>
```

Comment:
* [ ] FYI This creates a potentially expensive side-effect of importing. IMO acceptable, because it's only imported  by `scrivito_extensions`.